agent/utils/errors.go

```diff
@@ -67,6 +67,11 @@ func (me MultiErr) Error() string {
 	return strings.Join(ret, "\n")
 }
 
+func (me MultiErr) Append(err error) MultiErr {
```

Comment:
Yep, I wrote this before I realized that MultiErr could be initialized with a slice. Will delete.